docs/modules/reference/pages/glossary.adoc

```diff
@@ -41,16 +48,18 @@ Direct-discovered nodes:: Nodes that are added to {page-component-title} manuall
 Discourse:: An open-source forum platform that hosts https://opennms.discourse.group/[OpenNMS's community forums].
 There, users can find knowledge base articles, support, announcements, and general discussion about all things OpenNMS.
 
-xref:operation:provisioning/introduction.adoc#discovery-auto[Discovery]::
-A process that uses ICMP to find network devices and services in your network and then automatically adds them to monitoring.
+xref:operation:provisioning/introduction.adoc#discovery-auto[Discovery]:: A process that uses ICMP to find network devices and services in your network and then automatically adds them to monitoring.
 
-Docker:: An open-source container virtualization service that allows you to deploy application-level services without worrying about operating system maintenance.
+https://docs.docker.com/[Docker]:: An open-source container virtualization service that allows you to deploy application-level services without worrying about operating system maintenance.
 {page-component-title} components may optionally be deployed as Docker containers.
-(See https://docs.docker.com/[Docker].)
 
 Dominion:: The service on an OpenNMS core that controls the operation of Minions.
 (See xref:reference:configuration/minion-confd/minion-confd.adoc#dominion[Configuring Minion via Confd].)
 
+Drools:: A Business Rules Management System solution that supports the Java Rules Engine API standard.
```

Comment:
```suggestion
Drools:: A business rules management system that supports the Java Rules Engine API standard.
```
I'm not clear on if it's a solution that helps with business rules management systems, or if it is a system that helps with business rules management.

Reply:
I mostly grabbed this from the Wikipedia page, where the first sentence is,
> Drools is a business rule management system (BRMS) with a forward and backward chaining inference based rules engine, more correctly known as a production rule system, using an enhanced implementation of the Rete algorithm.